pom.xml

```diff
@@ -62,6 +62,13 @@
 
   <dependencies>
     <!-- regular dependencies -->
+    <dependency>
+      <groupId>org.jenkins-ci</groupId>
+      <artifactId>trilead-ssh2</artifactId>
+      <version>build217-jenkins-9</version>
```

Comment:
Agreed. I'll pick up the API plugin work as soon as I get to the bottom of any major outstanding Trilead issues.